src/org/ods/util/Project.groovy

```diff
@@ -1419,6 +1421,298 @@ class Project {
+            "description": "TC-CTR_M",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:TC_CTR_M"
+            ]
+        },
+        "PLTFMDEV-428": {
+            "key": "PLTFMDEV-428",
+            "name": "Functional / Requirements Testing Report_Manual",
+            "description": "C-FTR_M",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:FTR_M"
+            ]
+        },
+        "PLTFMDEV-427": {
+            "key": "PLTFMDEV-427",
+            "name": "Test Case Report",
+            "description": "TC-CTR",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:TC_CTR"
+            ]
+        },
+        "PLTFMDEV-426": {
+            "key": "PLTFMDEV-426",
+            "name": "Test Case Plan",
+            "description": "TC-CTP",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:TC_CTP"
+            ]
+        },
+        "PLTFMDEV-416": {
+            "key": "PLTFMDEV-416",
+            "name": "Configuration and Installation Testing Plan for DEV",
+            "description": "C-IVP for Dev",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:IVP"
+            ]
+        },
+        "PLTFMDEV-318": {
+            "key": "PLTFMDEV-318",
+            "name": "Technical Installation Plan",
+            "description": "C-TIP",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:TIP"
+            ]
+        },
+        "PLTFMDEV-317": {
+            "key": "PLTFMDEV-317",
+            "name": "Technical Installation Report",
+            "description": "C-TIR",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:TIR"
+            ]
+        },
+        "PLTFMDEV-24": {
+            "key": "PLTFMDEV-24",
+            "name": "Validation Summary Report",
+            "description": "C-VSR",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:VSR"
+            ]
+        },
+        "PLTFMDEV-23": {
+            "key": "PLTFMDEV-23",
+            "name": "Configuration and Installation Testing Report for P",
+            "description": "C-IVR for P",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:IVR_P"
+            ]
+        },
+        "PLTFMDEV-22": {
+            "key": "PLTFMDEV-22",
+            "name": "Technical Installation Report for P",
+            "description": "C-TIR for P",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:TIR_P"
+            ]
+        },
+        "PLTFMDEV-21": {
+            "key": "PLTFMDEV-21",
+            "name": "Configuration and Installation Testing Plan for P",
+            "description": "C-IVP for P",
+            "status": "DONE",
+            "labels": [
+                "LeVA_Doc:IVP_P"
+            ]
+        },
+        "PLTFMDEV-20": {
```

Comment:
@hrcornejo The new custom REST API endpoint does not deliver these data. We need to query it directly from the official Jira REST API using our Jira service. I suggest doing this in the load() method. Ideally, you manage to create exactly this structure, but coming from a different REST endpoint.

Reply:
I misunderstood. I'll fix it

Reply:
The JiraService is created in phaseInit depending on Project:
`    if (project.services?.jira) {
        withCredentials([ usernamePassword(credentialsId: project.services.jira.credentials.id, usernameVariable: "JIRA_USERNAME", passwordVariable: "JIRA_PASSWORD") ]) {
            registry.add(JiraService,
                new JiraService(
                    env.JIRA_URL,
                    env.JIRA_USERNAME,
                    env.JIRA_PASSWORD
                )
            )`
And the project is created formerly (calling load()) but now this will need JiraService:
`   def project = new Project(steps, git).load()`

So circular references...

Reply:
@hrcornejo then create a `Project.setJiraService()` that is invoked after `JiraService` has been initialized in phaseInit.groovy. Every method in Project that uses the services does an early return `if !this.jira`. Come on :)

Reply:
Done @metmajer